ui/src/PageOverlay.tsx

```diff
@@ -12,6 +12,7 @@ import { UserAnnotationLayer } from "./UserAnnotationLayer";
 interface PageProps {
   pageNumber: number;
   view: PDFPageView;
+  highlightedSymbols: string[];
```

Comment:
Same goes here as for the comment on the FindBarProps---wouldn't this make more sense to store in the ScholarReader's state, and then access it here through context?

Reply:
I believe in this case this would actually be okay, I can change it if you want but it will still need to be a prop on the findbar